include/mapnik/svg/svg_group.hpp

```diff
@@ -0,0 +1,48 @@
+/*****************************************************************************
```

Comment:
Maybe start using SPDX License headers? 
Instead the whole text using: 
`// SPDX-License-Identifier: LGPL-2.1-only`

See https://spdx.dev/ids/#how and https://spdx.org/licenses/LGPL-2.1-only.html

Reply:
Good call @mathisloge! I'll take a look, thanks.